src/librustc_yk_sections/emit_tir.rs

```diff
@@ -29,16 +34,26 @@ use std::io::Write;
 use std::error::Error;
 use std::cell::{Cell, RefCell};
 use std::mem::size_of;
+use std::convert::TryFrom;
 use rustc_data_structures::bit_set::BitSet;
 use rustc_data_structures::indexed_vec::{IndexVec, Idx};
-use rustc_data_structures::graph::dominators::DominatorFrontiers;
+use rustc_data_structures::graph::dominators::{Dominators, DominatorFrontiers};
+use rustc_data_structures::graph::WithSuccessors;
 use ykpack;
 use ykpack::LocalIndex as TirLocal;
+use ykpack::BasicBlockIndex as TirBasicBlockIndex;
 use rustc_data_structures::fx::FxHashSet;
 
 const SECTION_NAME: &'static str = ".yk_tir";
 const TMP_EXT: &'static str = ".yk_tir.tmp";
 
+/// The pre-SSA return value variable. In the pre-SSA TIR, we need the return value to reside at an
```

Comment:
The first half of this comment makes sense, but the second half is confusing. Possibly it's just because some sentences are in the wrong order?

Reply:
Can you expand on that -- it makes sense to me, but then I'm not a good critic for my own work.

Reply:
I think you just need to try rephrasing it. I can't make sense of the comment as a whole at the moment.